website/docs/interactors/4-write-your-own.md

```diff
@@ -3,28 +3,28 @@ id: write-your-own
 
-In this section, you will learn how to create a new Interactor for any interface and use it in your tests. We will start with a simple example for learning purposes, level up to a more complex example, and then cover common questions.
+In this section, you will learn how to write a new Interactor for any interface and use it in your tests. We will start with a simple example for learning purposes, level up to a more complex scenario, and then cover common questions.
 
 :::info Heads up
 
-There are new, exciting changes in the works for the `createInteractor()` API. Those changes will deprecate the syntax you see in the examples below, but the refactoring process will be very simple. We will output clear instructions in your console on how you can update to the new syntax so you can use Interactors today without worrying too much about tomorrow.
+There are new and exciting changes in the works for the `createInteractor()` API. Those changes will simplify the syntax you see in the examples below, but the refactoring process will be easy to adapt to. We will share clear instructions in your console on how you can update to the new syntax so you can use Interactors today without worrying about tomorrow’s easy-to-perform updates.
 
 :::
 
 
 ## Writing your first interactor
 
-In this tutorial, we will create our own `TextField` Interactor. Although BigTest has this type of interactor built-in, recreating it is a great way to learn all the pieces that make up an interactor, while using familiar examples.
+In this tutorial, we will create our own `TextField` Interactor. Although BigTest has this type of interactor built-in, recreating it is a great way to learn all the pieces that make up an interactor while using familiar examples.
 
-There are four things to decide:
+There are four things to decide when creating an interactor:
 1. What to name and label the interactor
 2. Which HTML element or elements to target
-3. The locator and filters, which helps users be able to narrow down the element they want to reference.
-4. Actions that a test should be able to `perform` on that element, like `click`
+3. The locator and filters, which helps users be able to narrow down the element they want to reference
+4. The action or actions that a test should `perform` when using the interactor (like `click`)
 
-Putting this information together, we can make this new Interactor:
+In this case, let's call our Interactor ‘MyTextField’, and we’ll label it as ‘my-textfield-interactor’ to distinguish it from the built-in one. Let’s say that MyTextField will target a regular input with a custom class `.my-input`, which means its HTML selector would be `input[type=text].my-input`. For locator we’ll use the input’s placeholder, and we’ll define a value filter. Finally, we’ll define a `fillIn` action for MyTextField interactor. Putting this together, we get the following definition of an Interactor:
```

Comment:
🤔 i missed this. i don't know why the `custom class '.my-input'` part was added. it's not implemented in the example. nor is the `MyTextField` interactor name.